frontend/packages/dev-console/integration-tests/features/gitops/linkArgoCD.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Argo CD Link
+    As a developer, I want a way to navigate to Argo CD from the Deploy Stages page of the OpenShift console so that I can get insight on the sync status of my application.
+
+
+    Background:
+        Given user is at the Administrator perspective
+        And user has installed Sealed Secrets Operator
+        And user has created SealedSecretController CR
+        And user has installed OpenShift Pipelines Operator
+        And user has installed Argo CD Operator
+        And user has installed GitOps Operator
+
+    
+    @regression, @manual
+    Scenario: External link to the ArgoCD Dashboard
+        Given user has created namespace "test-argocd"
+        And user has logged in to cluster using CLI
```

Comment:
Cant we login using web terminal operator

Reply:
We can do it but for that we have to add it in the background. Should I do it?